### PR TITLE
cleaned up f strings

### PR DESCRIPTION
**Pull request by m-kovalsky**

### CODE REVIEW

File: src/sempy_labs/_generate_semantic_model.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def create_semantic_model_from_bim(
         lro_wait=True,
     )
 
-    if response.status_code != 200:
+    if response.status_code not in [200, 201]:
```
Comment (eisber) [inline src/sempy_labs/_generate_semantic_model.py:129]: I'm surprised that you get both? don't have to change. just clarification.

File: src/sempy_labs/_helper_functions.py [hunk 2]
```diff
@@ -286,26 +286,34 @@ def get_direct_lake_sql_endpoint(dataset: str, workspace: Optional[str] = None)
...
-    dfE_filt = dfE[dfE["Name"] == "DatabaseQuery"]
-    expr = dfE_filt["Expression"].iloc[0]
+    with connect_semantic_model(
+        dataset=dataset, readonly=True, workspace=workspace
+    ) as tom:
+        sqlEndpointId = None
+        for e in tom.model.Expressions:
+            if e.Name == "DatabaseQuery":
+                expr = e.Expression
+                matches = re.findall(r'"([^"]*)"', expr)
```
Comment (eisber) [inline src/sempy_labs/_helper_functions.py:310]: you also want to find "" ? (so no characters in between quotes?

**Review by eisber — APPROVED** [review]

File: src/sempy_labs/_generate_semantic_model.py (hunk 1 above)
Comment (m-kovalsky) [reply src/sempy_labs/_generate_semantic_model.py:129]: i think it's 201 for this one but generally 201 or 200 are successes.

File: src/sempy_labs/_helper_functions.py (hunk 2 above)
Comment (m-kovalsky) [reply src/sempy_labs/_helper_functions.py:310]: it will just be a guid between the quotes

**Pull request merged**